flowauth/backend/flowauth/login.py

```diff
@@ -30,6 +30,18 @@ def signin():
     raise Unauthorized("Incorrect username or password.")
 
 
+@blueprint.route("/is_signed_in")
+def is_signed_in():
```

Comment:
Couldn't this be simplified by making it an `@login_required` route (so, you get a 401 if you aren't logged in, and a 200 `{"logged_in":True, "is_admin":bool(current_user.is_admin)}`?